portfolio/portfolio.go

```diff
@@ -444,3 +444,39 @@ func StartPortfolioWatcher() {
 func GetPortfolio() *Base {
 	return &Portfolio
 }
+
+// ExchangeSupported checks if exchange is supported by portfolio address
+func ExchangeSupported(exchange, address string) (ret bool) {
+	for x := range Portfolio.Addresses {
+		if Portfolio.Addresses[x].Address != address {
+			continue
+		}
+		exchangeList := strings.Split(Portfolio.Addresses[x].Address, ",")
+		if common.StringDataContainsInsensitive(exchangeList, address) {
+			return true
+		}
+	}
+	return
+}
+
+// ColdStorage checks if address is a cold storage wallet
+func ColdStorage(address string) (ret bool) {
```

Comment:
Could this be changed to `IsColdStorage`?

Reply:
:D done this is my ruby side coming out for bool returning methods

```ruby
portfolio.cold_storage?(":D")
```